README.md

```diff
@@ -293,18 +288,18 @@ For **∅ or more** and **1 or more** element(s) of the same type you can use th
 Union types are denoted with the pipe symbol, `|`:
 
 ```js
-(userInput: String|Number) => String|Number
+(userInput: String | Number) => String | Number
 ```
 
 ### Constructors
 
-Constructors in JavaScript require the `new` keyword. You can identify a constructor signature using the `new` keyword as if you were demonstrating usage:
+Constructors and classes are denoted using the `class` keyword:
```

Comment:
If you know JS, you can tell that classes are just a convenience glorified syntax.
As you can tell, I am not a big fan of `class` so I am a bit biased on this: Ill let you decide.
Is it an effort to be more compatible with TS?

If you wanna move on with that change the title will have to be changed.
You gotta make a clear distinction between the constructor statement of a class and what you call a "constructor".

You ought to provide at least 2 examples:
- constructor function
- class

Reply:
I think this deserves more thought. Thanks for your feedback on all this. =)